tests/test_access/tst_access_drogon_manage_login.py

```diff
@@ -121,10 +124,13 @@ def test_aggregations_fast(explorer: Explorer):
     cases = explorer.cases
     print("Number of cases: ", len(cases))
     assert len(cases) > 0
-    case = cases[0]
+    for c in cases:
+        if len(c.iterations) > 0 and len(c.surfaces) > 40:
+            case = c
+            break
     case_uuid = case.metadata.get("fmu").get("case").get("uuid")
```

Comment:
If there are no valid cases, so when `case=None`, will we get a useful error message?